crates/editor/src/editor.rs

```diff
@@ -6378,6 +6378,27 @@ impl Editor {
         self.reload(project, cx).detach_and_notify_err(cx);
     }
 
+    pub fn align_cursor(&mut self, _: &AlignCursor, cx: &mut ViewContext<Self>) {
+        self.transact(cx, |this, cx| {
+            let max_len = this
+                .selections
+                .all::<Point>(cx)
+                .iter()
+                .map(|s| s.start.column)
+                .max()
+                .unwrap_or(0);
+
+            let mut changes = Vec::new();
+            for s in this.selections.all::<Point>(cx).iter() {
+                let spaces = " ".repeat((max_len - s.start.column) as usize);
```

Comment:
The user may use tabs, is `" "` a good choice always?

Reply:
Also same question as above, 

> What if we select backwards? Then `start` will be after the `end`.


In addition to that, does this all make sense when `max_len == 0`? 
Looks like a noop, and `0 - something` may actually panic, which is super bad.
This edge case needs a test.